### DISCUSSION TITLE
Thanks for the fun.

### DISCUSSION THREAD

**Discussion by Rowsol** [TheBlueSavior/Project-Malice#4] [Show and tell]
When digging through the mod I noticed a couple enemies either had no Species tag or had one not correlating to the folder they were in. Just thought I'd mention it.

**Comment by TheBlueSavior** [comment]
Thanks so much for the videos! I'm glad you've enjoyed it so far, and I always enjoy seeing people play it. And thanks for bringing up the species tag, I'll get around to it sooner or later.

**Comment by Cormac1991** [comment]
Every day after work I get in bed and play this for an hour or so, been goin on a 3 years now. Really can't thank you enough for all the good times with these crazy monsters. Truly appreciate it, Thank you!

**Reply by TheBlueSavior** [reply]
Thanks for the kind words! I'm glad people have fun with it. I have no plans to stop working on PM, even if progress is slow. 3 more years from now I hope it'll be even better.

**Reply by c0mbolynch** [reply]
I would really appretiate it if we could disable the void monsters just like the fodders
